pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -350,7 +351,12 @@ public Representation post(Representation entity) {
 
           // 3/ Request is parsed by the handler which generates a
           // list of FileItems
-          items = upload.parseRequest(getRequest());
+          try {
+            items = upload.parseRequest(getRequest());
+          } catch (FileUploadBase.IOFileUploadException e) {
+            LOGGER.info("Exception uploading segment: " + e);
```

Comment:
This exception will be caused if the incoming request was bad, right? (say, the post did not conform to http protcool, or the connection was dropped and EOF received early). So, an INFO is fine, it seems. Why do you think it should be an ERROR?